scripts/update_configs_test.py

```diff
@@ -235,8 +235,9 @@ def mock_getpass(prompt):
             'since it is used in\n'
             '# the existing storage models for UserStatsModel.\n'
             'DASHBOARD_STATS_DATETIME_STRING_FORMAT = \'%Y-%m-%d\'\n')
-        expected_feconf_text = feconf_text.replace('None', '\'%s\'' % (
-            mailgun_api_key))
+        expected_feconf_text = feconf_text.replace(
```

Comment:
To be clear, I meant hardcode the whole thing (i.e. repeat the string, so that it serves as a "golden value").

Reply:
Ok, got it! Done.